test/apply.jl

```diff
@@ -182,95 +182,95 @@ facts("base element-wise operators on TimeArray values") do
+        @fact (cl .* ohlc).values                             --> cl.values .* ohlc.values
+        @fact (cl ./ ohlc).values                             --> cl.values ./ ohlc.values
+        @fact (cl .% ohlc).values                             --> cl.values .% ohlc.values
+        @fact (cl .^ ohlc).values                             --> cl.values .^ ohlc.values
+        @fact (cl .* (ohlc .> 200)).values                    --> cl.values .* (ohlc.values .> 200)
         @fact (basecall(cl, x->round(Int, x)) .* ohlc).values --> round(Int, cl.values) .* ohlc.values
         @fact_throws (ohlc["Open", "Close"] .+ ohlc) # One array must have a single column
     end
 
     context("correct comparison operations between TimeArray values and Int/Float (and viceversa)") do
-        @fact (cl .> 111.94).values[1]  --> false
-        @fact (cl .< 111.94).values[1]  --> false
-        @fact (cl .>= 111.94).values[1] --> true
-        @fact (cl .<= 111.94).values[1] --> true
-        @fact (cl .== 111.94).values[1] --> true
-        @fact (cl .!= 111.94).values[1] --> false
-        @fact (111.94 .> cl).values[1]  --> false
-        @fact (111.94 .< cl).values[1]  --> false
-        @fact (111.94 .>= cl).values[1] --> true
-        @fact (111.94 .<= cl).values[1] --> true
-        @fact (111.94 .== cl).values[1] --> true
-        @fact (111.94 .!= cl).values[1] --> false
-        @fact (ohlc .> 111.94).values[1,:]  --> [false true false false]
-        @fact (ohlc .< 111.94).values[1,:]  --> [true false true false]
-        @fact (ohlc .>= 111.94).values[1,:] --> [false true false true]
-        @fact (ohlc .<= 111.94).values[1,:] --> [true false true true]
-        @fact (ohlc .== 111.94).values[1,:] --> [false false false true]
-        @fact (ohlc .!= 111.94).values[1,:] --> [true true true false]
-        @fact (111.94 .> ohlc).values[1,:]  --> [true false true false]
-        @fact (111.94 .< ohlc).values[1,:]  --> [false true false false]
-        @fact (111.94 .>= ohlc).values[1,:] --> [true false true true]
-        @fact (111.94 .<= ohlc).values[1,:] --> [false true false true]
-        @fact (111.94 .== ohlc).values[1,:] --> [false false false true]
-        @fact (111.94 .!= ohlc).values[1,:] --> [true true true false]
+        @fact (cl .> 111.94).values[1]      --> false
+        @fact (cl .< 111.94).values[1]      --> false
+        @fact (cl .>= 111.94).values[1]     --> true
+        @fact (cl .<= 111.94).values[1]     --> true
+        @fact (cl .== 111.94).values[1]     --> true
+        @fact (cl .!= 111.94).values[1]     --> false
+        @fact (111.94 .> cl).values[1]      --> false
+        @fact (111.94 .< cl).values[1]      --> false
+        @fact (111.94 .>= cl).values[1]     --> true
+        @fact (111.94 .<= cl).values[1]     --> true
+        @fact (111.94 .== cl).values[1]     --> true
+        @fact (111.94 .!= cl).values[1]     --> false
+        @fact (ohlc .> 111.94).values[1,:]  --> [false, true, false, false]
```

Comment:
Yeah, 0.5 introduced "APL-style" array indexing, which drops unnecessary dimensions. I think there might be a Compat fix for that for 0.4.